flake.nix

```diff
@@ -300,6 +308,21 @@
               check_yaml_fmt
             '';
           };
+          check-scenario-visualiser-html = pkgs.writeShellApplication {
```

Comment:
This, plus the addition of the smoke test to `scripts/checks.sh`, might be a bit of cargo-culting -- I saw that all manner of other things were added to these two places, so there may be value in adding this one too. No idea if that's true; I don't need it for my own purposes.